src/app/helpers/isServer.js

```diff
@@ -0,0 +1,3 @@
+const isServer = () => !!(typeof process !== 'undefined' && process.versions);
```

Comment:
What is `!!` representing here?

Reply:
Feels like a horrible prettier thing which will make my skin crawl

Reply:
@pjlee11 If the value `(typeof process !== 'undefined' && process.versions)` is truthy, then this statement will return `true`.  A single `!` would return false, but a `!!` inverts that once more.